app/dao/inbound_sms_dao.py

```diff
@@ -31,6 +32,32 @@ def dao_get_inbound_sms_for_service(service_id, limit=None, user_number=None):
     return q.all()
 
 
+def dao_get_paginated_inbound_sms_for_service(
+    service_id,
+    user_number=None,
```

Comment:
user_number is never used for this method - should we get rid of it.

Reply:
ok